src/wasm/tomb_compat/wasm_mount.rs

```diff
@@ -141,7 +142,7 @@ impl WasmMount {
             .map_err(|e| format!("error while getting root cid for sync: {e}"))?;
 
         // todo(sstelfox): still need the following:
-        let valid_keys = vec![];
+        let verifying_keys = unlocked_drive.verifying_keys().await;
```

Comment:
I commented earlier on the change to banyanfs, but the filtering on which keys should be included should be done here, not deep in the drive functions. We want to include all keys except those that have the historical bit set. This means you'll need to include the AccessMask associated with each key as well.